client-spark/spark-2/src/main/scala/org/apache/spark/shuffle/celeborn/CelebornShuffleReader.scala

```diff
@@ -119,7 +119,7 @@ class CelebornShuffleReader[K, C](
         }
         metricsCallback.incReadTime(
```

Comment:
IMO we need to count `openStream` time into read time because it belongs to the shuffle read phase